### DISCUSSION TITLE
teslausb only for lockchime and boombox

### DISCUSSION THREAD

**Discussion by iFredLouzada** [marcone/teslausb#812] [General]
Hey, I am wondering if its possible to use teslausb only for music and boombox drives. Reason is I already have a 1tb solid ssd in the glovebox i use for steam and sentry and wanted to continue to use that for that, but wanted to plug in teslausb to the center console to have the ability to change my boombox and lockchime sounds using the convenience features of the teslausb stack.

I have commented out in the .conf file but it does not seem to take effect.

#export CAM_SIZE=30G
export MUSIC_SIZE=4G
export BOOMBOX_SIZE=100M

When i plug in the drive to windows i seem three drives: CAM, MUSIC and BOOMBOX and I can confirm that during tests the car wrote recent clips to telsausb.

**Comment by marcone** [comment]
That's not really a supported use case at the moment, though you can probably get close by specifying a really small CAM_SIZE, and then deleting the TeslaCam folder from the CAM drive.
